Target to generate vers.cpp

Hi, is there a way to generate vers.cpp without building whole project?
I see 'cpp/src/vers.cpp' target in Makefile, but when I try to execute it, I get the following:
```
user@host:~/open-zwave$ make cpp/src/vers.cpp
CPPFLAGS= make -C /home/user/open-zwave/cpp/build/ - cpp/src/vers.cpp
make[1]: Entering directory '/home/user/open-zwave/cpp/build'
make[1]: *** No rule to make target 'cpp/src/vers.cpp'.  Stop.
make[1]: Leaving directory '/home/user/open-zwave/cpp/build'
Makefile:32: recipe for target 'cpp/src/vers.cpp' failed
make: *** [cpp/src/vers.cpp] Error 2
```

--- a/Makefile
+++ b/Makefile
@@ -29,7 +29,7 @@ clean:
 	$(MAKE) -C $(top_srcdir)/cpp/examples/MinOZW/ -$(MAKEFLAGS) $(MAKECMDGOALS)
 
 cpp/src/vers.cpp:
-	CPPFLAGS=$(CPPFLAGS) $(MAKE) -C $(top_srcdir)/cpp/build/ -$(MAKEFLAGS) cpp/src/vers.cpp
+	CPPFLAGS=$(CPPFLAGS) $(MAKE) -C $(top_srcdir)/cpp/build/ -$(MAKEFLAGS) $(top_srcdir)/cpp/src/vers.cpp
 
 check: xmltest
 

